deployments/Dockerfiles/open_aea/Dockerfile

```diff
@@ -2,6 +2,8 @@ ARG AEA_VERSION=latest
 FROM valory/open-aea-user:${AEA_VERSION}
 ARG AEA_AGENT
 
+COPY openssl.cnf /etc/ssl
```

Comment:
I feel a little uncomfortable with this solution

Reply:
We can come up with the better solution, currently we have 2 ways if we don't want to follow this solution. First is to roll back to Ubuntu `20.04` or `18.04`, but there we won't be able to install `Python 3.10` since the base Ubuntu image for `20.04` or `18.04` does not have `Python 3.10` listed as official apt package we'll have to add it which I tried but failed for several reasons. Other way is to build `Python 3.10` from source while building the images. Should I create an issue for this?

Reply:
do you have an alternative solution in mind?